### ISSUE TITLE
Save and restore linked windows too

### ISSUE THREAD

**Issue by toddnni** [tmux-plugins/tmux-resurrect#134]
I sometimes link windows from a session to another, and currently after save and restore tmux-resurrect will duplicate the window instead of linking.

I tried to find out the best way to store the information in the resurrect file, but couldn't find one yet. The first idea was to store the linking information in window listing, but I read somewhere that linked windows are specific to a session and if one uses grouped sessions we might need different approach. 

I am able to make a pull request if I find the best approach, but currently I'm not confident with the concepts.

**Comment by toddnni** [comment]
I found out that "grouped sessions" works in my use case. So I can live without this. However, this would be nice to have as the current behavior is confusing.

**Comment by neerajbadlani** [comment]
+ 1 . Anything on this ?

**Comment by toddnni** [comment]
Sorry, but I'm not going to work on this.

**Comment by neverfox** [comment]
Yes, this is currently my tmux pain point because my daily-driver setup is three differently set up sessions all sharing at least one linked window. It doesn't survive reboot or kill-server and I have to re-do the linking each time. Interestingly, the linking does survive restore cycles done with the hotkeys. That seems to suggest that linking might be something that tmux can *do* but doesn't itself maintain any state about the linking after it has done it.

**Comment by bauen1** [comment]
Is there any progress on this ?
tmux-resurrect is one of the best tmux plugins, but this issue is mildy annoying.

**Comment by bruno-** [comment]
Hey, PR with a fix for this is welcome.
